src/modules/tomlpp.cppm

```diff
@@ -19,8 +20,8 @@ export namespace toml {
      * @brief The toml++ namespace toml::literals::
      */
     inline namespace literals {
-        using toml::literals::operator""_toml;
-        using toml::literals::operator""_tpath;
+        using TOML_NAMESPACE::literals::operator""_toml;
+        using TOML_NAMESPACE::literals::operator""_tpath;
     }
 
     using toml::array;
```

Comment:
@marzer Do you think

Reply:
Oh, hmmn, yeah. I completely forgot about all of that machinery.

In theory you should be able to simplify it by replacing 

```cpp
export namespace toml
{
	//...
}
```

with

```cpp
export TOML_NAMESPACE_START
{
	// ...
}
```

because `TOML_NAMESPACE_START` is shorthand for `namespace toml { inline namespace <internal abi namespace>`